src/widget/wtracktableview.cpp

```diff
@@ -1139,8 +1138,8 @@ void WTrackTableView::dropEvent(QDropEvent * event) {
                 // into the void we left, or down because of the new spot
                 // we're taking.
                 for (int i = 0; i < selectedRows.count(); i++) {
-                    if ((selectedRows[i] > movedRow) &&
-                        (destIndex.row() > selectedRows[i])) {
+                    if ((selectedRows[i] > movedRow) && (
+                        (destIndex.row() > selectedRows[i]) || destIndex.row() < 0 )) {
```

Comment:
It will simplify the code if you use a local like that:
(not verified) 
``` 
int destinationRow =  destIndex.row()
if (destinationRow < 0) {
    destinationRow = model()->rowCount()
}
```